.github/workflows/ci-build.yml

```diff
@@ -0,0 +1,18 @@
+name: Publish Docker image
+on:
+  release:
+    types: [published]
+jobs:
+  push_to_registry:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Push to Docker Hub
+        uses: docker/build-push-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          repository: mattrglobal/ion
```

Comment:
Is it possible to make this a variable? From my reading though it looks like github doesnt have repo variables..

Reply:
The only way we can pass variables through to a pipeline without mentioning the value in pipeline file is through secrets.Have made the repo name a secret.